config/kubernetes-sigs/sig-instrumentation/teams.yaml

```diff
@@ -46,3 +47,8 @@ teams:
     - DirectXMan12
     - s-urbaniak
     privacy: closed
+  prometheus-adapter-maintainers:
+    description: Write access to the prometheus-adapter repo
+    members:
+    - dgrisonnet
```

Comment:
Can we get this to match the OWNERS file?

Should be

```
- dgrisonnet
- lilic
- s-urbaniak
```

Reply:
Should we also add `s-urbaniak` to the maintainers, even though he is already in the admins list?

Reply:
Yes, he should be in both GitHub teams.

Reply:
Still missing Lili.